recipes/sasl/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "sasl" %}
+{% set version = "0.2.1" %}
+{% set md5 = "ca093d9a3d6f20b79b964a5e5add0202" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/8e/2c/45dae93d666aea8492678499e0999269b4e55f1829b1e4de5b8204706ad9/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  # entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - sasl = sasl:main
+    #
+    # Would create an entry point called sasl that calls sasl.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  number: 0
```

Comment:
You're going to need to specify the build script:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26